source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,242 @@
+.. _mdb-shell-help:
+
+===========================
+Access the |mdb-shell| Help
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+``mongosh`` provides additional help information in its help system.
+This document provides an overview of accessing this help
+information.
+
+.. _mdb-shell-help-command-line:
+
+Command Line Help
+-----------------
+
+To see the list of options and help for starting ``mongosh``, use the
+``--help`` option from the command line:
+
+.. code-block:: sh
+
+   mongosh --help
+
+Shell Help
+----------
+
+.. include:: /includes/admonitions/help-interchangeable.rst
+
+To see a basic list of commands you can run, in ``mongosh``, type
+``help``:
+
+.. code-block:: javascript
+
+   help
+
+.. _mdb-shell-help-db:
+
+Database Help
+-------------
+
+In ``monogsh``:
+
+- To see the list of databases on the server, use the ``show dbs``
+  command:
+
+  .. code-block:: javascript
+
+     show dbs
+
+  ``show databases`` is an alias for ``show dbs``.
+
+- To see the list of help for methods you can use on the ``db``
```

Comment:
[note for throughtout] You probably noticed this but this is really tough to scan with the content with each bullet so close together. I wonder if we can try putting a pipe `|` after each bullet to force a line break, which might space out the content a bit more.

Alternatively, we could try a definition list as in:

```
Database Help
-------------

Show List of Databases
  To see the list of databases on the server...

Show Help for a Specific Database Command
  To see help for a specific database method in ``mongosh``...

Show Additional Usage for a Database Command
  blahblahblah
```

Reply:
yeah, not a fan of the bulleted list. tried as definition lists.